package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "test": "hardhat test",
     "test:staging": "hardhat test --network goerli",
-    "lint": "solhint 'contracts/*.sol'",
+    "lint": "solhint 'contracts/**/*.sol'",
```

Comment:
This change seems unnecessary, as providing the base path--i.e. `contracts/*`--is sufficient.

Please correct me if I am overlooking this.

Reply:
see next line. I believe paths should be equal. Also `contracts/*` doesn't count `test` folder.